src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1947,6 +1993,17 @@
                     }
                 },
 
+                "translation": {
```

Comment:
>translation [](start = 17, length = 11)

Sorry, this was a typo in the eballot proposal. It should have been `translations` (all our array-valued property names are plural).

Reply:
Thanks Larry! that makes total sense :) yea it did sound strange

Reply:
noting this in the post-ballot changes list.

---
In reply to: [263051518](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263051518) [](ancestors = 263051518)

Reply:
(for post ballot list)

---
In reply to: [263052664](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263052664) [](ancestors = 263052664)